src/main/java/adris/altoclef/tasks/slot/ReceiveOutputSlotTask.java

```diff
@@ -34,25 +34,9 @@ protected void onStart(AltoClef mod) {
     protected Task onTick(AltoClef mod) {
         ItemStack inOutput = StorageHelper.getItemStackInSlot(_slot);
         ItemStack cursor = StorageHelper.getItemStackInCursorSlot();
-        if (!cursor.isEmpty() && !ItemHelper.canStackTogether(inOutput, cursor)) {
-            Optional<Slot> moveTo = mod.getItemStorage().getSlotThatCanFitInPlayerInventory(cursor, false);
-            if (moveTo.isPresent()) {
-                setDebugState("Moving cursor stack back");
-                return new ClickSlotTask(moveTo.get());
-            }
-            if (ItemHelper.canThrowAwayStack(mod, cursor)) {
-                setDebugState("Incompatible cursor stack, throwing");
-                return new ThrowCursorTask();
-            } else {
-                Optional<Slot> garbage = StorageHelper.getGarbageSlot(mod);
-                if (garbage.isPresent()) {
-                    setDebugState("Picking up garbage");
-                    return new ClickSlotTask(garbage.get());
-                } else {
-                    setDebugState("STUCK! Everything's protected.");
-                    return null;
-                }
-            }
+        boolean cursorSlotFree = cursor.isEmpty();
```

Comment:
We generalized clearing the cursor slot, which is what the removed code does. Replaced with EnsureFreeCursorSlotTask